src/AppInstallerCLICore/ExecutionContext.cpp

```diff
@@ -140,4 +140,9 @@ namespace AppInstaller::CLI::Execution
 
         m_executionStage = stage;
     }
+
+    void Context::SetThreadLocalThreadGlobals()
+    {
+        t_pThreadGlobals = &m_threadGlobals;

```

Comment:
`ThreadGlobals` is the only thing that should be directly touching the thread local variable.